index.html

```diff
@@ -113,61 +184,67 @@ <h1>
                 what you see and have a project you need coded,
                 don’t hestiate to contact me.</p>
             <button class="see-project-btn" style="width: 161px;">Get My Resume</button>
-            <img src="images/plsholder.png" class="crnr-img" alt="">
+            <img src="images/Frame.png" class="crnr-img" alt="">
```

Comment:
- [ ] [OPTIONAL] I have noticed that you are using the would be `background-image` as `img` tag 😎 . You should consider that this is a design image. Kindly, For [Semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) purposes, consider adding this image using `CSS` styling. 
  - If you notice in the image below, the stated `img` is also causing `overflow-x`. This breaks the application at different sizes which violates the responsiveness.
  -  Your design 😎 
  
![about image](https://user-images.githubusercontent.com/99394374/213858449-0ac7d042-de40-4933-b8bd-f6c25bdd4b92.PNG)